sdk/src/main/java/ly/count/android/sdk/ConnectionProcessor.java

```diff
@@ -215,7 +235,13 @@ private enum RequestResult {
             L.e("[ConnectionProcessor] urlConnectionForServerRequest, exception while calculating header field size: " + e);
         }
 
-        L.v("[Connection Processor] Using HTTP POST: [" + usingHttpPost + "] forced:[" + requestInfoProvider_.isHttpPostForced()
+        long headerFieldSizeTime = UtilsTime.getNanoTime() - pccTsStartHeaderFieldSize;
+        L.d("[ConnectionProcessor] urlConnectionForServerRequest, TIMING Header field size calculation took:[" + headerFieldSizeTime / 1000000.0d + "] ms");
```

Comment:
This is the code block where most of the request prep time occurs, I guess this header field length calculation should be refactored or not